channels/int.m3u

```diff
@@ -33,6 +33,8 @@ https://video.sekindo.com/blocktv/ch1/tracks-v3a1/mono.m3u8
 https://live.creacast.com/cna/smil:cna.smil/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/tXTjWgP.png" group-title="Religious",Christian Youth Channel
 http://media.smc-host.com:1935/cycnow.com/cyc2/chunklist_w258510680.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/en/thumb/7/7f/Disney_International_HD_logo.png/220px-Disney_International_HD_logo.png" tvg-country="IN" tvg-url="" group-title="Kids",Disney International HD
+http://stream-7b694ce7e.nmbs.top/m3u/Kids/10166da03f.m3u8
```

Comment:
The channel is broken. You should review it.

Reply:
@sultanfendonus for me it's working fine
<img width="758" alt="Captura de pantalla 2020-05-19 a las 14 55 51" src="https://user-images.githubusercontent.com/7253922/82323549-e57b2b80-99e0-11ea-9b29-44875f40bc60.png">